src/collections/resources/how-to-write-webassembly-filters-for-envoy/learn-how-to-write-wasm-filters.mdx

```diff
@@ -21,7 +21,7 @@ import { ResourcesWrapper } from "../Resources.style.js";
 <ResourcesWrapper>
   <div className="intro">
     <p>Learn more about WebAssembly's use within service mesh data planes in
-      <Link className="blog" to="/learn/service-mesh-books/the-enterprise-path-to-service-mesh-architectures-2nd-edition">The Enterprise Path to Service Mesh Archictures (2nd Edition)</Link> -  free book and excellent resource for anyone looking to understand WASM filters, Lua scripts, and other options available for extending the data plane.</p>
+      <Link className="blog" to="/learn/service-mesh-books/the-enterprise-path-to-service-mesh-architectures-2nd-edition"> The Enterprise Path to Service Mesh Archictures (2nd Edition)</Link> -  free book and excellent resource for anyone looking to understand WASM filters, Lua scripts, and other options available for extending the data plane.</p>
```

Comment:
Is this whitespace intentional?

Reply:
Yes it is, the link is missing a whitespace in front.

Reply:
@debo19 i don't understand. Normally, we don't want leading whitespace to be hyperlinked. What's the need here?

Reply:
@leecalcote oh okay I got what you mean. I was just fixing this. Working on another article PR, will add the space before the `<Link>` tag there.

<img width="496" alt="Screenshot 2022-02-22 at 6 40 38 PM" src="https://user-images.githubusercontent.com/85789734/155139107-b2a73f55-c396-43a0-b06f-3a89bd7c1621.png">